sdk/eventhub/Azure.Messaging.EventHubs/src/Consumer/LastEnqueuedEventProperties.cs

```diff
@@ -2,14 +2,16 @@
 // Licensed under the MIT License.
 
 using System;
+using System.ComponentModel;
+using Azure.Core;
 
 namespace Azure.Messaging.EventHubs.Consumer
 {
     /// <summary>
     ///   A set of information about the enqueued state of a partition, as observed by the consumer.
     /// </summary>
     ///
-    public struct LastEnqueuedEventProperties
+    public struct LastEnqueuedEventProperties : IEquatable<LastEnqueuedEventProperties>
```

Comment:
This was missed before GA.   Our guidelines ask that we make structs equatable if there is value for developers using the library to do direct comparisons.  In this case, it would be expected that developers perform an equality check when testing to see if the set of last enqueued event information for a partition has changed.   Monitoring these properties is a common scenario used for collecting metrics about the state of processing and understanding the "event backlog" within the service, often used to determine if throttling should be applied to senders and/or additional receivers be used.